test/test_torch.py

```diff
@@ -1139,6 +1139,57 @@ def test_pdist_norm(self):
                         self.assertEqual(expected.shape, actual.shape)
                         self.assertTrue(torch.allclose(expected, actual))
 
+    def test_squareform(self):
+        devices = ['cpu'] if not torch.cuda.is_available() else ['cpu', 'cuda']
+        for device in devices:
+            # smallest case []
+            vec = torch.empty(0, device=device)
+            sq = [[0]]
+            full_square = torch.tensor(sq, device=device)
+            full_square_ret = torch.squareform(vec)
+            self.assertEqual(full_square_ret, full_square, "{} did not equal expected result".format(full_square_ret))
+
+            # smallest case [1]
+            vec = torch.arange(1, 2, device=device)
+            sq = [[0, 1], [1, 0]]
+            full_square = torch.tensor(sq, device=device)
```

Comment:
what about tests for valid sizes, but invalid inputs, e.g. non-symmetric matrices?